sys/net/network_layer/ng_ndp/ng_ndp.c

```diff
@@ -28,6 +28,7 @@
 #include "vtimer.h"
 
 #include "net/ng_ndp.h"
+#include "net/ng_fib.h"
```

Comment:
Keeping the include inside the guard will trigger the same linker error btw. You'll just get another extra warning.